person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
- [Optional] @ahmedeid6842, I think since the person's id won't be provided during creation, we can assign random numbers to the id. One way to do it is my using the `rand` method